rest_framework_json_api/renderers.py

```diff
@@ -402,6 +407,11 @@ def extract_included(
                     included_cache[new_item["type"]][
                         new_item["id"]
                     ] = utils.format_field_names(new_item)
+                    meta = cls.extract_meta(field, serializer_data)
```

Comment:
Shouldn't this logic be moved into `build_json_resource_obj`? See comment above.